.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+on:
+  push:
+    branches: [develop, master]
+    tags: ["v*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
```

Comment:
The default Java version is the latest OpenJDK 8 HotSpot version via AdoptOpenJDK, it's possible to select another JDK as documented here https://github.com/olafurpg/setup-scala#usage For example to use JDK 11
```diff
      - uses: olafurpg/setup-scala@v7
+       with:
+         java-version: adopt@1.11
```